core/src/main/java/com/nisovin/magicspells/spells/PlayerMenuSpell.java

```diff
@@ -153,12 +161,33 @@ public boolean castFromConsole(CommandSender sender, String[] args) {
 		return true;
 	}
 
+	private ItemStack createItem(String path, String defaultName) {
+		ItemStack item = null;
+		if (isConfigSection(path)) {
+			MagicItem magicItem = MagicItems.getMagicItemFromSection(getConfigSection(path));
+			if (magicItem != null) item = magicItem.getItemStack();
+		} else {
+			MagicItem magicItem = MagicItems.getMagicItemFromString(getConfigString(path, ""));
+
+			if (magicItem == null) {
+				item = new ItemStack(Material.GREEN_WOOL);
+				ItemMeta itemMeta = item.getItemMeta();
+
+				itemMeta.setDisplayName("�6�t" + defaultName);
+				item.setItemMeta(itemMeta);
```

Comment:
You should use components (`ItemMeta#displayName`) instead of legacy strings (`ItemMeta#setDisplayName`) here.